Project.toml

```diff
@@ -16,11 +16,12 @@ Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
```

Comment:
Why do we need SparseArrays and we didn't need it before?

Reply:
I suppose that they fixed some type piracy with the release 1.4.1 : https://github.com/JuliaLang/julia/issues/30945 or https://github.com/JuliaLang/julia/issues/35899.